joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/AstCreatorHelper.scala

```diff
@@ -36,9 +36,12 @@ trait AstCreatorHelper { this: AstCreator =>
         parserNodeCache.addOne(json(ParserKeys.NodeId).num.toLong, pni)
         node match
           case CallExpr =>
-            json(ParserKeys.Fun)(ParserKeys.Obj).objOpt.map(obj => {
```

Comment:
You can revert these changes and take the latest updates from the master. A recent PR merge fixes this one as well as few more use cases.

Reply:
okay sure.

Reply:
done.